types/node/globals.d.ts

```diff
@@ -716,7 +716,7 @@ declare namespace NodeJS {
         isRaw?: boolean;
         setRawMode?(mode: boolean): void;
         _read(size: number): void;
-        _destroy(err: Error | null, callback: Function): void;
+        _destroy(err: Error | null, callback: (err?: Error) => void): void;
```

Comment:
```suggestion
        _destroy(err: Error | null, callback: (err?: Error | null) => void): void;
```
see `class Readable`